helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -356,6 +356,14 @@ protected void setDelayTimeInCluster(HelixZkClient zkClient, String clusterName,
     configAccessor.setClusterConfig(clusterName, clusterConfig);
   }
 
+  protected void setLastOnDemandRebalanceTimeInCluster(HelixZkClient zkClient,
```

Comment:
Could be static

Reply:
It's seems like `ZkTestBase` is only be used by it's subclasses. And all `protected` methods from `ZkTestBase` can be directly called by it's subclasses. So static is not needed?

Reply:
Discussed offline. Won't make any change.